### ISSUE TITLE
Feature request: support for direct bank transfers to customers

### ISSUE THREAD

**Issue by mmrko** [Tripletex/tripletex-api2#31]
Hi,

We have integrated our internal invoicing system to Tripletex and things have been running smoothly. We are currently investigating the possibility to automate paybacks to customers for the sales (provision) they generate on our platform. That is, we'd like to wire money (via the API), or alternatively via the UI, from our Tripletex integrated bank account (DNB) to our customers. Another accounting system we have an integration with (Procountor) supports this in their API (via `POST /payments/directbanktransfers`, https://dev.procountor.com/api-reference/). The closest thing I was able to find as per the [Tripletex API documentation](https://tripletex.no/v2-docs) was the concept of `supplierInvoice` but that doesn't seem to fit our use case (since the paybacks will need to be initiated by us).

Does this happen to be something you have in the pipeline and/or is there currently some other way to create outgoing payments to customers added to Tripletex? Thank you! :)

**Comment by tanettrimas** [comment]
It is possible in the UI/Web today (attached article here: https://tripletex.no/execute/docViewer?articleId=1022&language=0), but we do not have an api endpoint specifically for paying customers back via the bank integration.

**Comment by mmrko** [comment]
Hi, thanks for the swift reply! The UI approach seems to be for paying back on a specific invoice. Since we'd want to make a mere bank transfer, could we work around this by creating a "fake" invoice and refunding that in the aforementioned way? Or does the given invoice need to be paid first in full/partly in order to do any _remittering_ on it?

**Comment by tanettrimas** [comment]
I dont think that would work, because the function described in that document is primarily for paying back customers if they have paid too much on that invoice originally.

I checked our roadmap and I saw that we actually have plans to implement money transfers from Tripletex. Since we have not started development yet and this still is in the planning phase, I can't really give any more details here. But once we have more information to give out concering release/testing of this we can update this post. :)

**Comment by mmrko** [comment]
Hi! Any roadmap updates wrt supporting bank transfers via the API? Thanks :)

**Comment by andreasbhansen** [comment]
Also interested in this feature. Any updates to share? :)
